src/Raven.Server/Documents/Handlers/Processors/AbstractHandlerProxyReadProcessor.cs

```diff
@@ -32,6 +32,8 @@ public override async ValueTask ExecuteAsync()
         else
         {
             var command = CreateCommandForNode(nodeTag);
+            command.SelectedNodeTag = nodeTag;
```

Comment:
This means that we do have a command where we are not setting the `SelectedNodeTag` inside the 'CreateCommandForNode'. For me, it would be better to throw an exception here when SelectedNodeTag is null?

Reply:
TBH, I wonder if we need to pass that `SelectedNodeTag` in the `CreateCommandForNode` at all.
The change above will fit all cases, so why we need anything else?

Reply:
This was set to `protected internal` just for one specific case, do not spread the toxic further :)